Cargo.toml

```diff
@@ -56,8 +56,9 @@ categories = ["development-tools", "backend", "devtool"]
 [workspace.dependencies]
 actix-codec = "0.5"
 bincode = { version = "2.0.0-rc.2", features = ["serde"] }
-bytes = "1"
-tokio = { version = "1" }
+bytes = "1.7"
```

Comment:
Why do we prefer to be specific?

Reply:
Hmm, not sure actually. Just my habit :sweat_smile:

Reply:
I prefer the less specific, as we have lock anyway..